torntools/scripts/js/globalFunctions.js

```diff
@@ -1489,6 +1489,7 @@ const STORAGE = {
 				status_indicator: true,
 				block_ally_attacks: false,
 				notes: true,
+				show_chain_warning: true,
```

Comment:
Not sure if this should be on by default. I'll let @Mephiles decide.

Reply:
Nah, let's put it default: false.

Reply:
Done.